From eb4ba038ef0d7af55303ec951078783c80fb16c0 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 7 Feb 2023 14:46:53 +0100
Subject: [PATCH 279/954] feat(stm32mp2): increase BL2 size

With new developments (PMIC, DDR self-refresh and trusted boot), BL2
size needs to be increased to 184kB.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git cd50d774c9ae19103b512d000fc02cfab9dce5e6 feat(stm32mp2): increase BL2 size]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: Ib7e8eaaaa78ad95ab3ff8494ca979e64c8b293ad
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/289659
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index c0c1b1c4da..c9d8a5d3d3 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -147,7 +147,7 @@ enum ddr_type {
 #define STM32MP_BL31_SIZE		U(0x0001C000)	/* 112 KB for BL31 */
 
 #define STM32MP_BL2_RO_SIZE		U(0x00018000)	/* 96 KB */
-#define STM32MP_BL2_SIZE		U(0x0002B000)	/* 172 KB for BL2 */
+#define STM32MP_BL2_SIZE		U(0x0002E000)	/* 184 KB for BL2 */
 #define STM32MP_BL2_BASE		(STM32MP_SEC_SYSRAM_BASE + \
 					 STM32MP_SEC_SYSRAM_SIZE - \
 					 STM32MP_BL2_SIZE)
-- 
2.53.0

